recipes/coin-utils/all/conanfile.py

```diff
@@ -100,7 +120,7 @@ def package(self):
         self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
         with self._build_context():
             autotools = self._configure_autotools()
-            autotools.install()
+            autotools.install(args=["-j1",])
```

Comment:
I don't know why, but locally it was always a failure here (in my machine, defaults to `-j16`).